Sources/MapboxDirections/Directions.swift

```diff
@@ -439,8 +439,28 @@ open class Directions: NSObject {
      - returns: The data task used to perform the HTTP request. If, while waiting for the completion handler to execute, you no longer want the resulting skeleton routes, cancel this task.
      */
     @discardableResult open func refreshRoute(responseIdentifier: String, routeIndex: Int, fromLegAtIndex startLegIndex: Int = 0, completionHandler: @escaping RouteRefreshCompletionHandler) -> URLSessionDataTask? {
+        _refreshRoute(responseIdentifier: responseIdentifier,
+                      routeIndex: routeIndex,
+                      fromLegAtIndex: startLegIndex,
+                      currentRouteShapeIndex: nil,
+                      completionHandler: completionHandler)
+    }
+
+    @discardableResult open func refreshRoute(responseIdentifier: String, routeIndex: Int, fromLegAtIndex startLegIndex: Int = 0, currentRouteShapeIndex: Int, completionHandler: @escaping RouteRefreshCompletionHandler) -> URLSessionDataTask? {
```

Comment:
I assume we’re overloading the method instead of giving `currentRouteShapeIndex` a default value to avoid changing the method signature. We should document the new method and explain the difference between the two.